packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -88,7 +88,7 @@ export default {
     enableExtmap: false,
     experimental: {
       enableMediaNegotiatedEvent: false,
-      enableUnifiedMeetings: false
+      enableUnifiedMeetings: true
```

Comment:
Please remove from code so we can merge

Reply:
yeah this file was reverted in the last commit